src/app/zap-templates/zcl/data-model/chip/mode-select-cluster.xml

```diff
@@ -41,11 +41,12 @@ limitations under the License.
     <server init="false" tick="false">true</server>
     <description>Attributes and commands for selecting a mode from a list of supported options.</description>
     <!-- Base data types -->
-    <attribute side="server" code="0x0000" define="CURRENT_MODE"    type="INT8U"                                            writable="false" optional="false" reportable="true">CurrentMode</attribute>
-    <attribute side="server" code="0x0001" define="SUPPORTED_MODES" type="ARRAY" entryType="ModeOptionStruct" length="255"  writable="false" optional="false">SupportedModes</attribute>
-    <attribute side="server" code="0x0002" define="ON_MODE"         type="INT8U"                                            writable="true"  optional="true" default="255">OnMode</attribute>
-    <attribute side="server" code="0x0003" define="START_UP_MODE"   type="INT8U"                                            writable="false" optional="false">StartUpMode</attribute>
-    <attribute side="server" code="0x0004" define="MODE_DESCRIPTION" type="CHAR_STRING"                                     writable="false" optional="false" length="32">Description</attribute>
+    <attribute side="server" code="0x0000" define="MODE_DESCRIPTION" type="CHAR_STRING"                                     writable="false" optional="false" length="32">Description</attribute>
+    <attribute side="server" code="0x0001" define="STANDARD_NAMESPACE" type="ENUM16"                                        writable="false" optional="true"  default="65535">StandardNamespace</attribute>
+    <attribute side="server" code="0x0002" define="SUPPORTED_MODES" type="ARRAY" entryType="ModeOptionStruct" length="255"  writable="false" optional="false">SupportedModes</attribute>
+    <attribute side="server" code="0x0003" define="CURRENT_MODE"    type="INT8U"                                            writable="false" optional="false" reportable="true">CurrentMode</attribute>
+    <attribute side="server" code="0x0004" define="START_UP_MODE"   type="INT8U"                                            writable="false" optional="false">StartUpMode</attribute>
```

Comment:
This is nullable in the spec.  Why is it not nullable here?

Also, this is optional in the spec.  Why is it not optional here?